### DISCUSSION TITLE
Avatars missing from Hubs cloud

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#3424] [Hubs Cloud Questions] [answered]
On Hubs Cloud all my avatars have disappeared. In Hubs Admin panel I can see them all listed as featured. In the room _Featured_, _MyAvatars_ and _Newest_ are all empty. Avatars previously worked. I have tried importing custom files as glb and as .pack files and importing mozilla defaults. All work on hubs.mozilla.com just not on Hubs cloud. 

Is there a fix??? I have an event Friday would like some avatars. thnx

**Comment by robinkwilson** [comment, marked as answer]
Can you provide screenshots of the admin panel + in your room + open developer menu in browser > console / network tabs, see if there's any failing requests? Thanks!

**Reply by BonzoMartini** [reply]
..Sorry its too late, delete the whole thing and redo it. It is fine now (apart from the server crashing during the live event)

**Reply by robinkwilson** [reply]
No worries! Well... Not ideal but glad you're up and running. 

Not great about the crash -- if you know how to reproducible steps for that, please make an issue: https://github.com/mozilla/hubs/issues

I'm going to close this for now.
